frontend/src/app/choreography/choreography.component.ts

```diff
@@ -75,13 +85,8 @@ export class ChoreographyComponent implements OnInit {
   isVoiceSynthesisOn = false;
   tempo = 8;
   areNotesShown = false;
-
-  constructor() {
-  }
-
-  ngOnInit(): void {
-    this.choreography.frames[0].subframes[0].grid = this.generateGrid();
-  }
+  availableGroupTypes: GroupType[] = ['two', 'three', 'four'];
```

Comment:
**suggestion:** store this somewhere as an array so we don't end up repeating it in components